modules/account-lib/test/resources/algo/index.ts

```diff
@@ -38,27 +38,27 @@ export const accounts = {
+      'jKNhbXTNJxClY2xvc2XEIPxRzY5iGKGjjaR+0AIw8FgIFu0TujMDrF3rkRVIkIAlo2ZlZc1V8KJmdgGjZ2VurHRlc3RuZXQtdjEuMKJnaMQgSGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiKibHZkomx4xCCXDKhSkaeCFr/a4vTlGdynAwSMwrYTZ7G+zr0BrJ74o6Rub3RlxARub3Rlo3JjdsQgPUAXw+hDiVqStwqnTRt+vJyYLM8uxJaMwM1V8Sr0Zgyjc25kxCDXWpgBgrEKt9VL/tPJZAc6DuFy89qmIyWvAhpo9wdRGqR0eXBlo3BheQ==',
     signed:
-      '82a3736967c44070146fd2a30ef31153e200c4b9df29e046035fe5ad89341770074dd532068d03236c907f1a03cdbb834cca3e517d87f14551f48c2569592e8a7a339b5d33fd0ea374786e8ba3616d74cd2710a5636c6f7365c420fc51cd8e6218a1a38da47ed00230f0580816ed13ba3303ac5deb911548908025a3666565cd55f0a2667601a367656eac746573746e65742d76312e30a26768c4204863b518a4b3c84ec810f22d4f1081cb0f71f059a7ac20dec62f7f70e5093a22a26c7664a46e6f7465c4046e6f7465a3726376c4203d4017c3e843895a92b70aa74d1b7ebc9c982ccf2ec4968cc0cd55f12af4660ca3736e64c420d75a980182b10ab7d54bfed3c964073a0ee172f3daa62325af021a68f707511aa474797065a3706179',
+      'gqNzaWfEQBNuiJOd4DX48r1j58j648aePWemKIfQoolycRgNGbWBS4veklsCtLwx6/XDf4gd37VBVzHlA+uSelt65RyYvw6jdHhujKNhbXTNJxClY2xvc2XEIPxRzY5iGKGjjaR+0AIw8FgIFu0TujMDrF3rkRVIkIAlo2ZlZc1V8KJmdgGjZ2VurHRlc3RuZXQtdjEuMKJnaMQgSGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiKibHZkomx4xCCXDKhSkaeCFr/a4vTlGdynAwSMwrYTZ7G+zr0BrJ74o6Rub3RlxARub3Rlo3JjdsQgPUAXw+hDiVqStwqnTRt+vJyYLM8uxJaMwM1V8Sr0Zgyjc25kxCDXWpgBgrEKt9VL/tPJZAc6DuFy89qmIyWvAhpo9wdRGqR0eXBlo3BheQ==',
     // signed with account1 only
     halfSigned:
-      '82a46d73696783a67375627369679282a2706bc420d75a980182b10ab7d54bfed3c964073a0ee172f3daa62325af021a68f707511aa173c440bf584b2a9ec266c7809aaa0f848687c78d0145dc5a8967e14456d11697c72e8b9ad6b8eecbe8aea41fcf384d05a6af54e9d0928fba7d29edbdf27d9ef3c9a50081a2706bc420fc51cd8e6218a1a38da47ed00230f0580816ed13ba3303ac5deb911548908025a374687202a17601a374786e89a3616d74cd2710a3666565cd55f0a2667601a367656eac746573746e65742d76312e30a26768c4204863b518a4b3c84ec810f22d4f1081cb0f71f059a7ac20dec62f7f70e5093a22a26c7664a3726376c4203d4017c3e843895a92b70aa74d1b7ebc9c982ccf2ec4968cc0cd55f12af4660ca3736e64c4205a874762a5e8f55e0a19995ae37d60a6355a26b830386eca3639ec3928660c2ca474797065a3706179',
+      'gqRtc2lng6ZzdWJzaWeSgqJwa8Qg11qYAYKxCrfVS/7TyWQHOg7hcvPapiMlrwIaaPcHURqhc8RALU7Eycpgkq6o5Ib/jYKr/FBIDMrEtK3DW+YaKS87+AAYuiMGaDBf2WjABqWFvld0EQ9TmkMqQSw/8h/jFAzhC4GicGvEIPxRzY5iGKGjjaR+0AIw8FgIFu0TujMDrF3rkRVIkIAlo3RocgKhdgGjdHhuiqNhbXTNJxCjZmVlzVXwomZ2AaNnZW6sdGVzdG5ldC12MS4womdoxCBIY7UYpLPITsgQ8i1PEIHLD3HwWaesIN7GL39w5Qk6IqJsdmSibHjEICo7EN+p/8tIgA7K8AKIc06cjObUr/dFU6vXmcXso/n1o3JjdsQgPUAXw+hDiVqStwqnTRt+vJyYLM8uxJaMwM1V8Sr0Zgyjc25kxCBah0dipej1XgoZmVrjfWCmNVomuDA4bso2Oew5KGYMLKR0eXBlo3BheQ==',
```

Comment:
Why is the transaction smaller when you add lease param? Shouldn't be bigger?

Reply:
Same for others

Reply:
Is it due encoding format update?

Reply:
Yes I think so,  since base64 generates shorter strings than hex